<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>presenter</class>
 <widget class="QMainWindow" name="presenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>458</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>presenter</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>380</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" rowspan="3">
        <widget class="QGraphicsView" name="graphicsView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="select_colony">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Other Colonies..</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="current_colony">
         <property name="minimumSize">
          <size>
           <width>461</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Current Colonie: Lithium-Stollen 13-A</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QGroupBox" name="group_1">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>295</width>
           <height>163</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>295</width>
           <height>163</height>
          </size>
         </property>
         <property name="title">
          <string>                                  Program</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="3" rowspan="3">
           <widget class="QPlainTextEdit" name="textfield">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>169</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>169</width>
              <height>70</height>
             </size>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="undoRedoEnabled">
             <bool>false</bool>
            </property>
            <property name="lineWrapMode">
             <enum>QPlainTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="plainText">
             <string/>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
            <property name="maximumBlockCount">
             <number>1</number>
            </property>
            <property name="centerOnScroll">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="delete_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>⌫</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="up">
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/pfeile/pfeile/pfeil_oben.png</normaloff>:/pfeile/pfeile/pfeil_oben.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="right">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/pfeile/pfeile/pfeil_rechts.png</normaloff>:/pfeile/pfeile/pfeil_rechts.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="wait">
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>⦿</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="comboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>169</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>169</width>
              <height>24</height>
             </size>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Repetitions</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="left">
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>28</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/pfeile/pfeile/pfeil_links.png</normaloff>:/pfeile/pfeile/pfeil_links.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="down">
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/pfeile/pfeile/pfeil_unten.png</normaloff>:/pfeile/pfeile/pfeil_unten.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>up</zorder>
         <zorder>left</zorder>
         <zorder>right</zorder>
         <zorder>textfield</zorder>
         <zorder>comboBox</zorder>
         <zorder>wait</zorder>
         <zorder>down</zorder>
         <zorder>delete_2</zorder>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QGroupBox" name="group_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>295</width>
           <height>141</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>295</width>
           <height>141</height>
          </size>
         </property>
         <property name="title">
          <string>                             Mission Progress</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="start">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>121</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="single_step">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>121</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Single-Step</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="auto_run">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Auto-Run</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="stop">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="cancel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>select_colony</tabstop>
  <tabstop>start</tabstop>
  <tabstop>up</tabstop>
  <tabstop>left</tabstop>
  <tabstop>right</tabstop>
  <tabstop>single_step</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>textfield</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>wait</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
